os-eap64-launch/added/launch/ha.sh

```diff
@@ -42,6 +44,20 @@ function check_view_pods_permission() {
     fi
 }
 
+function validate_dns_ping_settings() {
+  if [ "x$OPENSHIFT_DNS_PING_SERVICE_NAME" = "x" ]; then
+    >&2 echo "WARNING: Environment variable OPENSHIFT_DNS_PING_SERVICE_NAME undefined. Clustering will be unavailable. Please refer to the documentation for configuration."
+  fi
+}
+
+function validate_ping_protocol() {
+  if [ "$1" = "openshift.KUBE_PING" ]; then
+    check_view_pods_permission
+  elif [ "$1" = "openshift.DNS_PING" ]; then
+    validate_dns_ping_settings
+  fi
```

Comment:
should we perhaps warn about unrecognized protocol? (use case: typos and copy'n'paste errors)